.changeset/spicy-clouds-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+'@siafoundation/explored-types': minor
+---
+
+explored-types: fixed webpack-related empty export warning
```

Comment:
No need for the `explored-types:` prefix in the actual message, its already categorized. Also, start capitalized and end with a period. eg: https://github.com/SiaFoundation/web/releases/tag/renterd%400.58.0

Reply:
Fixed! I think I was confusing/merging changeset formatting with something you recently talked about wanting to see in commit messages.

Reply:
Yes, that was regarding commit messages, but for now we are still following conventional commits, you have an error:
<img width="905" alt="Screenshot 2024-08-26 at 2 39 44 PM" src="https://github.com/user-attachments/assets/cca9b7e4-2bb3-481f-b7ac-64582f3c8183">
should be `fix:`.

Reply:
The current style for commits is eg:
`fix: explorer-types add type only import for core types`